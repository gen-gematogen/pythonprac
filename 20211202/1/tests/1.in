class C(metaclass=dump):
	def __init__(self, name):
		print(f"Hello, my name is {name}")
		self.name = name

	def boo(self):
		print(f"{self.name} is scared!")

c = C('Suzy')
c.boo()
