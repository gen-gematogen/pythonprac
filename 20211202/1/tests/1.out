__init__: ('Suzy',), {}
Hello, my name is Suzy
boo: (), {}
Suzy is scared!
